management/src/test/java/org/ehcache/docs/ManagementTest.java

```diff
@@ -99,8 +99,10 @@ public void usingManagementRegistry() throws Exception {
 
         // hit count is a sampled stat, for example its values could be [0,0,3,4].
         // In the present case, only the last value is important to us , the cache was eventually hit 4 times
-        int mostRecentIndex = onHeapStore_Hit_Count.getValue().length - 1;
-        onHeapHitCount = onHeapStore_Hit_Count.getValue()[mostRecentIndex].getValue();
+        if (onHeapStore_Hit_Count.getValue().length > 0) {
```

Comment:
@anthonydahanne there is actually a better fix.   The test can still pass when the index is -1.  It just needs more time to gather the statistics.  So I wouldn't use an assert that will make the test fail.   Just wrap that check in an if statement.  I posted a similar fix below that I need to check in for the ManagementTest